packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -37,12 +41,13 @@ class _AccountPictures extends StatelessWidget {
                 padding: const EdgeInsetsDirectional.only(start: 8.0),
                 child: Semantics(
                   container: true,
-                  child: Container(
-                  padding: const EdgeInsets.only(left: 8.0, bottom: 8.0),
-                    width: 48.0,
-                    height: 48.0,
-                    child: picture,
-                 ),
+                  child: Padding(
+                    padding: const EdgeInsets.only(left: 8.0, bottom: 8.0),
```

Comment:
Is this padding following any material design spec? If not, can it be customizable? (I'm pretty new to to this widget)

Reply:
Well that should be for an other PR...